src/flows/Onboarding/OnboardingInvite.tsx

```diff
@@ -37,8 +37,8 @@ export function OnboardingInvite({
       const response = await employmentInviteMutationAsync({
         employment_id: onboardingBag.employmentId,
       });
-      if (response.data?.data) {
-        onSuccess?.(response.data.data);
+      if (response.data) {
+        onSuccess?.(response.data as SuccessResponse);
```

Comment:
`onSuccess` callback should also support promises, don't you think? I know we would have to change this in every flow, so this can be done in a different PR.